awx/api/generics.py

```diff
@@ -964,14 +966,19 @@ def post(self, request, *args, **kwargs):
         if hasattr(new_obj, 'admin_role') and request.user not in new_obj.admin_role.members.all():
             new_obj.admin_role.members.add(request.user)
         if sub_objs:
+            # store the copied object dict into memcached, because it's
+            # often too large for postgres' notification bus
+            # (which has a default maximum message size of 8k)
```

Comment:
Is that not a problem for other stuff, like job events?

and how does this work on a cluster where a different instance may be performing the deep copy?

Reply:
This limitation is specific to *only* dispatcher tasks (which use postgres listen/notify).

Job events use redis.

Reply:
> and how does this work on a cluster where a different instance may be performing the deep copy?

see: https://github.com/ansible/awx/pull/6034/files#diff-9d4ea1dd908b35fb92eaede4bd10bb46R2856

We made it so that this task is routed to dispatchers on the *same* instance as uwsgi (they don't get distributed to other nodes).